modules/ossm-add-project-member-roll-resource-console.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-add-project-member-roll-recourse-console_{context}"]
+= Adding or removing projects from the service mesh using ServiceMeshMemberRoll resource with the web console
+
+* You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+
+* The `ServiceMeshMemberRoll` resource is deleted when its corresponding `ServiceMeshControlPlane` resource is deleted.
+
+*  If the control plane runs on an infrastructure node, skip this task.
+
+.Prerequisites
+* An installed, verified {SMProductName} Operator.
+* An existing `ServiceMeshMemberRoll` resource.
+* Name of the project with the `ServiceMeshMemberRoll` resource.
+* Names of the projects you want to add or remove from the mesh.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Click the *Project* menu and choose the project where your `ServiceMeshControlPlane` resource is deployed from the list, for example `istio-system`.
```

Comment:
```suggestion
. Click the *Project* menu and choose the project where your `ServiceMeshControlPlane` resource is deployed from the list. For example, `istio-system`.
```

Reply:
To be consistent with other examples in this PR.

Reply:
done